submissions/description/broken_access_control/idor/template.md

```diff
@@ -1,19 +1,19 @@
 # Insecure Direct Object Reference (IDOR)
 
-## Overview of the Vulnerability 
+## Overview of the Vulnerability
 
-Insecure Direct Object Reference (IDOR) occurs when there are no access control checks in place to verify if the request of an object within the application can be viewed by the person requesting access. Depending on the type of IDOR within an application, an attacker could perform the following actions:
+Insecure Direct Object Reference (IDOR) occurs when there are no access control checks in place to verify if the request of an object within the application can be viewed by the person requesting access. Given which type of IDOR within an application, an attacker could perform the following actions:
```

Comment:
I'm wondering if this line can be clearer:
> Insecure Direct Object Reference (IDOR) occurs when there are no access control checks in place to verify if the request of an object within the application can be viewed by the person requesting access.

IDOR doesn't always have to be a request for an object, it can be also updating an object; inline with the following actions (which covers them off well) and I feel this overview should include this possibility.

Reply:
@vortexau Thanks, I think i've got the wording tweaked right now to reflect the various actions

Reply:
Excellent update!